studio/src/pages/[organizationSlug]/apikeys.tsx

```diff
@@ -347,7 +347,10 @@ const CreateAPIKeyDialog = ({
                               : "Select graphs"}
                           </Button>
                         </DropdownMenuTrigger>
-                        <DropdownMenuContent align="start" className="">
+                        <DropdownMenuContent
+                          align="start"
+                          className="scrollbar-custom max-h-[calc(var(--radix-dropdown-menu-content-available-height)_-24px)] overflow-y-auto"
```

Comment:
Can we enforce a max height? On the video, the dropdown is very high.

Reply:
Done